app/controllers/api/v1/admin/user/invoices_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::Admin::User::InvoicesController < Api::V1::BaseController
+  before_action :authenticate_user!, :load_user
+
+  def index
+    authorize :invoice, :index?
+    @invoice = @user.timelogs.date_range(Date.parse(invoice_params.to_s).beginning_of_month, Date.parse(invoice_params.to_s).end_of_month)
+    render json: [@invoice, @user, @user.salary_type, @user.salary, @user.count_day_offs_this_month]
```

Comment:
this is not a good example of response, because on front end you need to work with array indexes and nobody knows what data[3] is.